src/components/PageBlocks/CalendarBase.vue

```diff
@@ -342,6 +346,10 @@ export default {
       }
       return 'auto'
     },
+
+    refresh () {
+      this.config.datesRender()
```

Comment:
Would changing :key in this component reload the component?

Reply:
It only reloads the fullcalendar component so it does not recache this computed property to force an update without calling the `config.datesRender()` function.